src/components/cms/BotBuilder/BotBuilderPages/DesignViews/CodeView.js

```diff
@@ -18,11 +19,30 @@ import 'brace/theme/solarized_light';
 import 'brace/theme/terminal';
 import 'brace/ext/searchbox';
 
+const Container = styled.div`
+  margin-bottom: 40px;
+`;
+
+const AceEditor = styled(_AceEditor)`
```

Comment:
We should make a shared component for Editor. Please open an issue for it.

Reply:
#2433